src/main/scala/com/linkedin/spark/datasources/tfrecord/TFRecordFileReader.scala

```diff
@@ -23,7 +23,7 @@ object TFRecordFileReader {
     val recordType = options.getOrElse("recordType", "Example")
 
     val inputSplit = new FileSplit(
-      new Path(new URI(file.filePath)),
```

Comment:
can you remove the unused import `import org.apache.hadoop.fs.Path`?

Reply:
@mizhou-in thanks for the feedback. Of course, I'll take care of these items.